driver-core/src/main/com/mongodb/AwsCredential.java

```diff
@@ -38,12 +39,12 @@
      *
      * @param accessKeyId the non-null access key ID that identifies the temporary security credentials.
      * @param secretAccessKey the non-null secret access key that can be used to sign requests
-     * @param sessionToken the non-null session token
+     * @param sessionToken the session token, which may be null
```

Comment:
This is a small but backwards-compatible API change which allows us to use this class internally as well as externally, and doesn't really harm the external users.